front/src/reducers/osrdconf/osrdConfCommon/index.ts

```diff
@@ -176,11 +178,17 @@ export function buildCommonConfReducers<S extends OsrdConfState>(): CommonConfRe
     updateTimetableID(state: Draft<S>, action: PayloadAction<S['timetableID']>) {
       state.timetableID = action.payload;
     },
-    updateElectricalProfileSetId(
+    updateElectricalProfileSetID(
```

Comment:
Can you keep updateElectricalProfileSetId ? We would like to remove the `ID` in maj of the code base

Reply:
Ho, by looking at the code base, I was thinking the opposite, that's why I changed it. 
I will reverse it, thanks.